server.js

```diff
@@ -1,3 +1,7 @@
+const debug = (process.env.NODE_DEBUG || "").split(",");
+if (debug.indexOf("riff") >= 0) {
```

Comment:
looks like the condition is inverted

```suggestion
if ((process.env.NODE_DEBUG || "").split(",").includes("riff")) {
```

Another option is the enable debug if NODE_DEBUG is set at all

```suggestion
if (process.env.NODE_DEBUG) {
```

Reply:
oopsie, quick change at the end of day :|